src/examples/ParkingLot/Input/ParkingLot.xml

```diff
@@ -1795,34 +1795,34 @@
 		</mesh>
 	</object>
 	<object id="4">
-		<metadata type="name">Ground</metadata>
+		<metadata type="name">area</metadata>
 		<mesh>
 			<vertices>
 				<vertex>
 					<coordinates>
-						<x>59789.1</x>
-						<y>50004.5</y>
+						<x>-111608</x>
```

Comment:
the room coordinates file was not updated. please re-run the simulation so that all outputs are updated accordingly